packages/components/src/components/autocomplete/autocomplete.scss

```diff
@@ -3,6 +3,7 @@
 
 .dc-autocomplete {
     width: 100%;
+    margin-bottom: 3.2rem;
```

Comment:
Why was this brought back?

Reply:
when resolving conflict i saw two version, margin-bottom with `rem` and with `em`. so, i chose rem, guess, it should be removed at all then 👍